### DISCUSSION TITLE
ReferenceError: is_op_policy_updating_enabled is not defined this.process

### DISCUSSION THREAD

**Discussion by justatwo** [rabbitmq/rabbitmq-server#7721] [Questions] [answered]
RabbitMQ 3.11.10
Erlang 25.1.1 
when i go to the tab Admin-Policies，i get error"ReferenceError: is_op_policy_updating_enabled is not defined ReferenceError: is_op_policy_updating_enabled is not defined at Array.process (eval at compile (http://192.168.222.131:15672/js/ejs-1.0.min.js:1:6654), :228:49) at EJS.render (http://192.168.222.131:15672/js/ejs-1.0.min.js:1:1885) at format (http://192.168.222.131:15672/js/main.js:1164:21) at http://192.168.222.131:15672/js/main.js:486:24 at with_reqs (http://192.168.222.131:15672/js/main.js:1146:9) at http://192.168.222.131:15672/js/main.js:1142:17 at req.onreadystatechange (http://192.168.222.131:15672/js/main.js:1237:17) "

![error-rabbitmq](https://user-images.githubusercontent.com/67728375/226823388-be1ea032-0dd9-40ef-8f6e-6d0735886f52.png)

**Comment by justatwo** [comment]
i  have update the version of rabbitmq and erlang.But this error still exist.i have no idea now.

**Comment by cihatsolak** [comment, marked as answer]
the error message you are seeing indicates that there is an issue with the code used to render the Admin-Policies tab in RabbitMQ. One possible cause could be a compatibility issue between the version of RabbitMQ you are using and the web browser you are using to access it. You could try accessing RabbitMQ with a different web browser or upgrading to a newer version of RabbitMQ to see if the issue is resolved. If the problem persists, you may need to troubleshoot any configuration issues or consult the RabbitMQ documentation for support.

**Reply by michaelklishin** [reply]
It's just a browser cache that needs clearing.

**Reply by justatwo** [reply]
thank you all.i change to use firefox browser.it is ok now.
i hava cleared my browser cache.this problem still exist.the browser version is  Microsoft Edge 111.0.1661.51.
![image](https://user-images.githubusercontent.com/67728375/227078263-2083d05e-30ee-4c8c-bb11-725f2090b65a.png)

**Comment by michaelklishin** [comment]
Clearing your browser cache should help.
